apps/backoffice-v2/src/pages/Entity/components/Case/Case.Actions.tsx

```diff
@@ -66,7 +66,7 @@ export const Actions: FunctionComponent<IActionsProps> = ({
               <span className={`mr-[8px] text-sm leading-6`}>Status</span>
               <Badge
                 variant={tagToBadgeData[tag].variant}
-                className={ctw(`text-sm font-bold`, {
+                className={ctw(`whitespace-nowrap text-sm font-bold`, {
```

Comment:
What's this change for?

Reply:
On small screen button text is wrapping to next line and break how it looks.

Reply:
Will test flex-shrink and min-width later on.